app/src/main/java/com/genymobile/pr/bus/LoadingErrorEvent.java

```diff
@@ -0,0 +1,23 @@
+package com.genymobile.pr.bus;
+
+import android.support.annotation.Nullable;
+
+import retrofit.Response;
+
+public class LoadingErrorEvent {
+    private final Response response;
+    private final boolean networkError;
+
+    public LoadingErrorEvent(boolean networkError, @Nullable Response response) {
```

Comment:
Would'n it be better to have 2 different kind of errors instead of having a boolean param ?